website/moneybirdsynchronization/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from thaliawebsite import settings
+
+from members.models import Member
+
+class Contact(models.Model):
+    member = models.OneToOneField(
+        Member,
+        on_delete=models.CASCADE,
+        verbose_name=_("member"),
+        null=True,
+        blank=True,
```

Comment:
I think adding a related name `moneybird_contact` would be nice, so you can access it via `member.moneybird_contact`